seantan22/theia
# VSX: Add 'Install Another Version...' Command

#### What it does


- Supports the ability to install any compatible version of an user-installed extension, provided that the extension is available in the Open VSX Registry.

https://user-images.githubusercontent.com/46289281/114419822-c3067980-9b81-11eb-9155-6556d10b7919.mp4

#### How to test


1. Open the extensions view and locate the `Installed` view of all user-installed extensions.
2. Pick an extension --> Click the `manage` gear icon --> Click `Install Another Version...` command. Note that the command is disabled for any extension not available in the Open VSX Registry.
3. Select a compatible version to install from the Quick Pick dropdown.
4. Observe that the previous version is uninstalled and replaced with the newly selected version.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: seantan22 <sean.a.tan@ericsson.com>
